modules/shapefile/src/lib/util.js

```diff
@@ -0,0 +1,20 @@
+export const LITTLE_ENDIAN = true;
```

Comment:
This is a random assortment of things. I don't think `parseHeader` should go into a generic util.js

Reply:
I only put it there because it's identical for parsing both shp and shx headers. 

for the little/big endian I can just define it in each file